cupy/padding/pad.py

```diff
@@ -1,321 +1,738 @@
+import cupy
```

Comment:
Import statements must be organized into three parts: standard libraries, third-party libraries, and internal imports. [H306]
Could you move this line after numpy imports?